### PR TITLE
KHP3-4780: (feat) add mch clinical views for anc, labour and delivery, and postn…

### PR DESCRIPTION
**Pull request by ojwanganto**
…atal care

## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

This PR adds clinical views for MCH mother services - anc, labor and delivery, and postnatal care. 

## Screenshots

image

## Related Issue

*None.*

### CODE REVIEW

**Review by donaldkibet — CHANGES_REQUESTED** [review]
Thanks @ojwanganto Great start 🚀. Let some suggestions.

**Comment by ojwanganto** [comment]
image
Updated UI. Changed from tabbed to content switcher

File: packages/esm-patient-clinical-view-app/src/esm-mch-app/constants.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
+export const mnchEncounterType_UUID = '3eb24129-9f7d-4372-a14e-c2cf577e5db1';
```
[the remark this answers is not in the document]
Comment (ojwanganto) [reply packages/esm-patient-clinical-view-app/src/esm-mch-app/constants.ts:1]: I will clean this up better in a separate PR

File: packages/esm-patient-clinical-view-app/src/data-table/o-table.component.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+} from '@carbon/react';
+import styles from './o-table.scss';
+import { LazyCell } from '../lazy-cell/lazy-cell.component';
+
+interface TableProps {
+  tableHeaders: any;
+  tableRows: any;
+}
+
+export const OTable: React.FC<TableProps> = ({ tableHeaders, tableRows }) => {
```
[the remark this answers is not in the document]
Comment (ojwanganto) [reply packages/esm-patient-clinical-view-app/src/data-table/o-table.component.tsx:24]: This is not entirely obs table. It is a wrapper around Encounter and observations.

**Review by donaldkibet — APPROVED** [review]

**Comment by donaldkibet** [comment]
Greate! 🤝 Thanks @ojwanganto

**Pull request merged**